google/cloud/spanner/samples/samples.cc

```diff
@@ -2221,6 +2221,114 @@ void CustomRetryPolicy(std::vector<std::string> argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void CustomInstanceAdminPolicies(std::vector<std::string> argv) {
+  if (argv.size() != 1) {
+    throw std::runtime_error("custom-instance-admin-policies <project-id>");
+  }
+  //! [custom-instance-admin-policies]
+  namespace spanner = ::google::cloud::spanner;
+  [](std::string const& project_id) {
+    // An instance admin client is controlled by three policies. The retry
+    // policy determines for how long the client will retry transient failures.
+    auto retry_policy = spanner::LimitedTimeRetryPolicy(
+                            /*maximum_duration=*/std::chrono::minutes(25))
+                            .clone();
+    // The backoff policy controls how long does the client wait between
```

Comment:
s/wait/will wait/?

[Most of these are repeated in the DatabaseAdmin part.]

Reply:
Fixed this one and the remaining comments.